Flag to disable cumulativity of Prop (#18828)
coq/coq: doc/sphinx/proof-engine/vernacular-commands.rst

   This :term:`boolean attribute` is similar to the :flag:`Universe Checking` flag, but on a per-declaration basis.
   Disable universe checking locally with ``bypass_check(universes)``.

.. flag:: Cumulative Prop

>    This :term:`flag` can be used to control whether `Prop` is in
>    cumulativity with the predicative universes (`Set` and instantiations of `Type`).
>    It is on by default.

Reviewer: ```suggestion
   Controls whether `Prop` is cumulative for the predicative universes (`Set` and
   instantiations of `Type`).  It is on by default.
```

If correct.  "`Prop` is in cumulativity with the predicative universes" reads very strangely and unclear.
